### DISCUSSION TITLE
Positive & potential negative values using bar shape

### DISCUSSION THREAD

**Discussion by RobinBertilsson** [airbnb/visx#1774] [Q&A] [answered]
Using a "plain bar chart", would it be possible to display negative values according to image below?
I've found half-ish examples of this, however - most referring to outdated D3 documentation etc...

Would love a working example 🙏 

<img width="903" alt="image" src="https://github.com/airbnb/visx/assets/4040035/b1e37a94-9b13-4dba-93eb-e4f59f768f83">

**Comment by williaster** [comment, marked as answer]
Hey @RobinBertilsson 👋  thanks for checking out `visx`. Here's a [sandbox](https://codesandbox.io/embed/xenodochial-dream-wvyqqj?fontsize=14&hidenavigation=1&theme=dark) with an example of something like this modified from our [documentation bargroup example](https://airbnb.io/visx/bargroup).

<img width="589" alt="image" src="https://github.com/airbnb/visx/assets/4496521/82852f0b-5c33-4aea-8ee6-3a0ba73e90ae">

Here's also a related discussion if useful https://github.com/airbnb/visx/issues/662. Hope that helps!
